simulation.py

```diff
@@ -277,23 +277,29 @@ def epsilon_to_png(self):
                     h5_file=self.eps_file + ':data', 
                     output_file=defaults.temporary_epsh5)
         log.info("calling: {0}".format(callstr))
-        if not sp.call(callstr.split(), cwd=self.workingdir):
-            # no error, continue:
-            dct = dict(self.__dict__, h5_file=defaults.temporary_epsh5)
-            # save dielectric to png:
-            if self.geometry.is3D:
-                callstr = (defaults.epsh5topng_call_3D % dct,
-                           defaults.epsh5topng_call_3D_cross_sect % dct)
-            else:
-                callstr = (defaults.epsh5topng_call_2D % dct,)
-            retcode = 0
-            for s in callstr:
-                if not retcode:
-                    log.info("calling: {0}".format(s))
-                    retcode = retcode or sp.call(s.split(), 
-                                                 cwd=self.workingdir)
-        else:
+        
+        try:
+            sp.call(callstr.split(), cwd=self.workingdir)
+        except OSError as err:
+            log.warning('The following OSError occurred while calling mpb:' +
+                        '\n\t{}\n'.format(err) + 
+                        '\tMaybe mpb is not properly insalled.')
```

Comment:
typo: insTalled ;)
I'm curios: How exactly is MPB not properly installed? Is mpb-data missing, because you only installed mpb-mpi? Or is h5utils missing (which is not MPB)? I'd rather be more specific here.